### PR TITLE
[Doc] Add initial variable and fragments

### PR DESCRIPTION
**Pull request by Justinterest**
Now the docs website support variable and fragments render.
And writer can write docs using variable and fragments

### variables
- variables file in `docs/variables.json`, you can add variable in this file, like `{"version": "v0.9.0","title":"Taichi"}`
- then you can using variable on your docs like `{{var.version}}`, remember that no space inside of `{{}}`

### fragments
- the fragments is in `docs/fragments/` folder, you can create fragment file like `test/company-desc.md`
- then you can using that on your docs like `{{fragments/test/company-desc.md}}`

### CODE REVIEW

**Comment by Vissidarte-Herman** [comment]
@Justinterest So, if I have multiple variables, I shall separate them with a comma?
Gentle reminder, don't forget to sign the CLA.:)

**Comment by Justinterest** [comment]
>

**Comment by Justinterest** [comment]
The variables file is a JSON file, all variables in key-value format, like
``` json
{
  "title": "Taichi",
  "version": "v0.9.0"
}
```
you can add new variable to the json, if you want add `faqlink`, after add would like
``` json
{
  "title": "Taichi",
  "version": "v0.9.0",
  "faqlink": "/faq"
}
```

**Review by Vissidarte-Herman — APPROVED** [review]
LGTM!

File: docs/variable.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (rexwangcc) [inline docs/variable.json:1]:
```suggestion
{}

```

**Review by rexwangcc — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**